src/components/list-item-text-icon/list-item-text-icon.tsx

```diff
@@ -25,28 +25,34 @@ const styles: StyleRulesCallback<CssRules> = (theme: ArvadosTheme) => ({
     hasMargin: {
         marginLeft: `${theme.spacing.unit}px`,
     },
+    fixFontSize: {
+        fontSize: '1.25rem'
+    }
 });
 
 export interface ListItemTextIconDataProps {
     icon: IconType;
     name: string;
     isActive?: boolean;
     hasMargin?: boolean;
+    fixFontSize?: boolean;
 }
 
 type ListItemTextIconProps = ListItemTextIconDataProps & WithStyles<CssRules>;
 
 export const ListItemTextIcon = withStyles(styles)(
     class extends React.Component<ListItemTextIconProps, {}> {
         render() {
-            const { classes, isActive, hasMargin, name, icon: Icon } = this.props;
+            const { classes, isActive, hasMargin, name, icon: Icon, fixFontSize } = this.props;
             return (
                 <Typography component='span' className={classes.root}>
                     <ListItemIcon className={classnames({
                             [classes.hasMargin]: hasMargin,
                             [classes.active]: isActive
                         })}>
-                        <Icon />
+                        <Icon className={classnames({
+                            [classes.fixFontSize]: fixFontSize
```

Comment:
Instead of setting fixFontSize class could we here have `Icon style={{fontSize: "${fontSize}rem"}}` ? Then we could replace `fixFontSize` with `iconSize` and everyone could easily set any icon size.